xarray/backends/h5netcdf_.py

```diff
@@ -350,6 +348,38 @@ class H5netcdfBackendEntrypoint(BackendEntrypoint):
     For more information about the underlying library, visit:
     https://h5netcdf.org
 
+    Parameters
+    ----------
+    group:
+    mode:
+    format:
+    lock:
+    autoclose:
+    driver: str or None, optional
+        Name of the driver to use. Legal values are None (default,
+        recommended), "core", "sec2", "direct", "stdio", "mpio", "ros3".
+    driver_kwds: Mapping or None, optional
+        Additional driver options. See h5py.File for more infos.
+    invalid_netcdf : bool or None, optional
+        Allow writing netCDF4 with data types and attributes that would
+        otherwise not generate netCDF4 files that can be read by other
+        applications. See :ref:`invalid netcdf` for more details.
+    phony_dims: {"sort", "access"} or None, optional
+        Change how variables with no dimension scales associated with
+        one of their axes are accessed.
+
+        - None: raises a ValueError (default)
+
+        - "sort": invent phony dimensions according to netCDF behaviour.
+        Note, that this iterates once over the whole group-hierarchy.
+        This has affects on performance in case you rely on laziness
+        of group access.
+
+        - "access": defer phony dimension creation to group access time.
+        The created phony dimension naming will differ from netCDF behaviour.
+    decode_vlan_strings: bool
```

Comment:
```suggestion
    decode_vlen_strings: bool
        Return VLEN string data as str instead of bytes. Defaults to True.
```

This is still relevant as the h5py default is to return bytes and xarray default is expecting str. The decision was to be flexible on that, hence the kwarg.

Reply:
Thanks!